src/UI/CsobControl.php

```diff
@@ -100,7 +100,13 @@ public function pay()
 		if ($payment->getOriginalPayId()) {
 			$response = NULL;
 			try {
-				$response = $this->client->paymentRecurrent($payment);
+				if ($this->client->isRecurrentPaymentSupported()) {
+					$response = $this->client->paymentRecurrent($payment);
+
+				} else {
+					$initResponse = $this->client->paymentOneclickInit($payment);
```

Comment:
are you sure this isn't coliding with `$this->client->createPayment()` ? Seems weird you'd have to call both "inits".